src/hotspot/share/include/jvm.h

```diff
@@ -1159,6 +1159,12 @@ JVM_VirtualThreadUnmountBegin(JNIEnv* env, jobject vthread, jboolean last_unmoun
 JNIEXPORT void JNICALL
 JVM_VirtualThreadUnmountEnd(JNIEnv* env, jobject vthread, jboolean last_unmount);
 
+/*
+ * Value types support.
```

Comment:
Value types? This is supporting the core reflection work isn't it?

Reply:
I fixed this in the second commit.